docs/from-scratch/3-customization.md

```diff
@@ -93,24 +93,29 @@ Next, add some CSS and override the markup so that the component looks like this
 The important part to remember here is that the original component can be imported via a `script` tag's `src` parameter. If you intend to override Javascript, though, you have to choose a different method – see below.
 
 ::: tip Parse HTML content
-Rembember that HTML content shipped by RTE should be parsed. To do so, use the `<html-parser>` component.
+Rembember that HTML content shipped by RTE should be parsed. To do so, use the `<t3-html-parser>` component.
+:::
+
+::: warning Override by registration
+To make it work you have to register your new component as the global one. Please look step [2.2](#_2-2-register-your-component)
 :::
 
 **Override component logic**
 
-If you want to change the Javascript of the original component, then you first have to setup the component as explained in #2.1. However, now the origianl component is imported via the `import` statement. That way we can employ the `extends` method on `export default {}` and add or override Javascript:
+If you want to change the Javascript of the original component, then you first have to setup the component as explained in #2.1. However, now the original component is imported via the `import` statement. That way we can employ the `extends` method on `export default {}` and add or override Javascript:
```

Comment:
"That way we can employ the `extends` method". I think we should replace the 'method' world with 'option' as we are talking about Vue options API.